modules/ximgproc/CMakeLists.txt

```diff
@@ -1,2 +1,2 @@
 set(the_description "Extended image processing module. It includes edge-aware filters and etc.")
-ocv_define_module(ximgproc opencv_core opencv_imgproc opencv_calib3d opencv_imgcodecs WRAP python java)
+ocv_define_module(ximgproc opencv_core opencv_imgproc opencv_calib3d opencv_imgcodecs opencv_tracking  WRAP python java)
```

Comment:
We prefer to keep dependency list minimal - adding tracking here, this would block adding ximgproc as dependency into tracing module.

Do we really need this dependency?

Reply:
Thank you for that comment. In fact tracking is not needed anymore.